elementary/monitor/api/tests/schema.py

```diff
@@ -62,6 +62,7 @@ class TestMetadataSchema(BaseModel):
     description: Optional[str] = None
     result: dict
     configuration: dict
+    test_tags: List[str] = Field(default_factory=list)
```

Comment:
maybe we should add this to `SourceFreshnessMetadataSchema` as well (just empty always). just because we are planning to use this field for all items in the table

Reply:
Will do